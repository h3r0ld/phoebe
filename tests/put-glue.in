start map.txt
hardworking-little-robot-random off
put-stain G
quit
