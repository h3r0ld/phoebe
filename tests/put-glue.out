Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
Cell(22, 8): GameObject: GlueStain
Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
